testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDatabricksWithHttpMetastore.java

```diff
@@ -0,0 +1,150 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.deltalake;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tempto.BeforeMethodWithContext;
+import io.trino.tempto.assertions.QueryAssert;
+import io.trino.testng.services.Flaky;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.UUID;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_DATABRICKS_HTTP_HMS;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_ISSUE;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_MATCH;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestDatabricksWithHttpMetastore
+        extends BaseTestDeltaLakeS3Storage
+{
+    private String unityCatalogCatalogName;
+    private String unityCatalogExtLocation;
+
+    @BeforeMethodWithContext
+    @Override
+    public void setUp()
+    {
+        super.setUp();
+        unityCatalogCatalogName = requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_NAME"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_NAME");
+        unityCatalogExtLocation = requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_EXT_LOC"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_EXT_LOC");
+    }
+
+    @Test(groups = DELTA_LAKE_DATABRICKS_HTTP_HMS)
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testBasicTableReadWrite()
+    {
+        String schemaName = "testBasicConnection_" + randomNameSuffix();
+        String tableName = schemaName + ".testTable";
+        String tableLocation = createRandomTablePath(tableName);
+        try {
+            createSchemaInDelta(schemaName);
+            onDelta().executeQuery(format("CREATE TABLE %s.%s (c1 int, c2 string) using delta location 's3://%s/%s'", unityCatalogCatalogName, tableName, bucketName, tableLocation));
+            onDelta().executeQuery(format("INSERT INTO %s.%s values (1, 'one')", unityCatalogCatalogName, tableName));
+            List<QueryAssert.Row> expectedSchemas = ImmutableList.of(
+                    row(schemaName.toLowerCase(Locale.ROOT)));
+            assertThat(onTrino().executeQuery("SHOW SCHEMAS FROM delta"))
+                    .contains(expectedSchemas);
```

Comment:
Replace with `containsOnly` and don't create list here and in following places.

Reply:
containsOnly cannot be applied here in case of schemas because we will have other schemas like `information_schema` as well in addition to the expected schema. I will replace with containsOnly to other places where it applies.

Reply:
done